berserk/clients/teams.py

```diff
@@ -73,3 +73,19 @@ def accept_join_request(self, team_id: str, user_id: str) -> None:
         """
         path = f"/api/team/{team_id}/request/{user_id}/accept"
         self._r.post(path)
+
+    def get_team(self, team_id: str) -> Dict[str, Any]:
```

Comment:
Can you update the ``README.rst`` with your new methods ? :)

Reply:
Should I do this manually, or is it automatically generated while we create docs?

Reply:
They need to be added manually.